### ISSUE TITLE
Failing to load multiple files using static loader , using { "file" :

### ISSUE THREAD

**Issue by aykutyararbas** [angular-translate/angular-translate#923]
```js
//given
var options= {
            "files" :  [{
                prefix: 'i18n/gen-',
                suffix: '.json'
            }, {
                prefix: 'i18n/gen-aykut-',
                suffix: '.json'
            }]
    };
//when
$translateProvider.useStaticFilesLoader(genevaOptions);

//Then
Either gen-en.json or gen-aykut-en.json loaded


//Modifying the load function helped load multiple files 

  $http(angular.extend({
        url: [
          file.prefix,
          file.key,
          file.suffix
        ].join(''),
        method: 'GET',
        params: ''
      }, options.$http)).success(function (data) {
        deferred.resolve(data);
      }).error(function (data) {
        deferred.reject(file.key);
      });

//You can check within plunkr
http://plnkr.co/edit/UnWwcifNEy0AQKW0yPgW?p=preview
```

Thanks and regards

Aykut Yararbas~

**Comment by tspaeth** [comment]
The plnkr isn't working correctly at all.

But anyway - the options for the standard static files loader are currently only able to take an option input object as defined in https://github.com/angular-translate/angular-translate/blob/master/src/service/loader-static-files.js

```
@param {object} options Options object, which gets prefix, suffix and key.
```

and not an Array of Objects. So if you need a totally different approach - yes you have to write a custom loader / parser.

**Comment by aykutyararbas** [comment]
Thanks
Aykut~

**Comment by janhmueller** [comment]
Actually, Aykut is correct. You should be able to pass a files array to the loader. Its both in the docs ["Alternatively, if you have multiple translation files in distinct locations, you may instead supply an array of files to the loader" + code example] (http://angular-translate.github.io/docs/#/guide/12_asynchronous-loading) and in the code, if you look a bit closer. Aykut's changes seem to fix it.
Best, Jan

**Comment by knalli** [comment]
There is a bug in the processing.

**Comment by knalli** [comment]
Fixed in latest 2.6.1
